references/recognition/README.md

```diff
@@ -64,3 +64,13 @@ Feel free to inspect the multiple script option to customize your training to yo
 ```python
 python references/recognition/train_pytorch.py --help
 ```
+## Using custom fonts
+If you want to use your own custom fonts for training, make sure the font is installed on your OS.
+Do so on linux by copying the .ttf file to the desired directory with: ```sudo cp custom-font.ttf /usr/local/share/fonts/``` and then running ```fc-cache -f -v``` to build the font cache.
+
+Keep in mind that passing fonts to the training script will only work with the WordGenerator which will not augment or change images from the dataset if it is passed as argument. If no path to a dataset is passed like in this command ```python3 doctr/references/recognition/train_pytorch.py crnn_mobilenet_v3_small --vocab french --font "custom-font.ttf"```  only then is the WordGenerator "triggered" to create random images from the given vocab and font.
+
+Running the training script should look like this for multiple custom fonts:
+```shell
+python references/recognition/train_pytorch.py crnn_vgg16_bn --epochs 5 --font "cusomt-font-1.ttf,custom-font-2.ttf"
```

Comment:
small typo here "custom-font-1.ttf" :)

Reply:
Hello, 
Not very familiar with how I can fix this after the pull request was merged, do I need to create a new one?

Reply:
Hi @carl-krikorian short answer: yep :)